ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift

```diff
@@ -237,6 +248,10 @@ open class InputViewController: UIInputViewController, KeymanWebDelegate {
         }
       }
     }
+    
+    self.swallowBackspaceTextChange = true
+    // afterDelay 0.01 is not sufficient!  0.02 seems to be enough for an actual delay to register.
+    perform(#selector(self.clearSwallowFlag), with: nil, afterDelay: 0.02)
```

Comment:
Yeah, this is ugly.  I don't see another way around the issue at present, though - we need the wait to ensure that `textDidChange` doesn't incorrectly wipe the embedded KMW's context due to desync.  There are no calls that occur later to allow us to consistently refresh the context after the fact.

We can't just ignore `textDidChange` because it's needed to detect legitimate changes of the context.  (Selections, caret movement, copy/paste, etc.)